code/obj/item/assembly/misc_assemblies.dm

```diff
@@ -719,6 +719,35 @@ Contains:
 	var/obj/item/new_target = new /obj/item/pipebomb/bomb(src)
 	src.set_up_new(null, new_trigger, new_applier, new_target)
 
+
+/////////////////////////////////////////////////// timer pipebomb ////////////////////////////////////
+// These do include tiny syndicate pipebombs and engineering pipbombs
+
+
+/obj/item/assembly/timer_ignite_pipebomb
+	secured = TRUE
+	var/pipebomb_path = /obj/item/pipebomb/bomb
+	var/bomb_strenght = null
+
+/obj/item/assembly/timer_ignite_pipebomb/New()
+	..()
+	var/obj/item/device/timer/new_trigger = new /obj/item/device/timer(src)
+	new_trigger.time = 3 SECONDS //have a time already set up and don't blow up unsuspecting users on use
```

Comment:
```suggestion
	new_trigger.time = 5 SECONDS //have a time already set up and don't blow up unsuspecting users on use
```
To match the old pipebomb timer. I've pushed a fix to make timers actually accurate at low durations so it should match up nicely.